infrastructure/backup/README.md

```diff
@@ -0,0 +1,70 @@
+# Backup system - Velero
+
+[Velero](https://velero.io/) is a backup & migration cloud native software design to perform disaster recovery and migrate resources across cluster.
+
+The main purpose of Velero in CrownLabs is to _mitigate_ the following eventualities:
+- failed/buggy updates
+- unintentional resources deletion
+- catastrophical physical cluster destruction/loss of all masters(*)
+a backup system can be of use.
+
+(*) this would require backups to be stored in a physically different/distant location relatively to the cluster.
+
+Notes:
+- saving backups on the cluster itself could help to mitigate unintentional deletions but it's not a great strategy, since deletion of volumes or related resources could lead in deletion of backups themselves.
+
+## How has been deployed
+
+The backup solution chosen for CrownLabs is based on [Velero](https://velero.io/). 
+
+Velero can store backups of Kubernetes resources and possibly associated volumes on several cloud native/proprietary storage solutions (such as AWS, GCP, Azure, VMWare, OpenStack, etc.).
+
+In case of full bare-metal environments, it's possible to use [Minio](https://min.io/), an open source, Kubernetes-native onbject storage solution which is compliant with Amazon's S3 API. This provides a valid storage target for Velero, provided that the mounted data path is not being stored on the cluster itself.
```

Comment:
```suggestion
In case of full bare-metal environments, it is possible to use [Minio](https://min.io/), an open source, Kubernetes-native object storage solution which is compliant with Amazon's S3 API. This provides a valid storage target for Velero, provided that the mounted data path is not being stored on the cluster itself.
```

Better not to use abbreviations in technical documentation